lib/libc/termios/lib_tcloopback.c

```diff
@@ -0,0 +1,53 @@
+/****************************************************************************
```

Comment:
Should we make library file only for tc?? I think ioctl is enough, library is unnecessary.

Reply:
ok. you mean I should remove this file from here and should call ioctl from iotbus??
if this is the case then I did so to maintain symmetry because tc files are also there in lib.
please let me know what you think now??